src/js/port_handler.js

```
@@ -300,6 +330,11 @@ PortHandler.updateDeviceList = async function (deviceType) {
                     ports = await serial.getDevices("webbluetooth");
                 }
                 break;
+            case "capacitorserial":
+                if (this.showCapacitorOption || true) {
```

Biome (2.1.2)

[error] Unexpected constant condition.



(lint/correctness/noConstantCondition)